repositories.yaml

```diff
@@ -1149,3 +1149,25 @@ applications:
     channels:
       - v1_name: treeherder
         app_id: treeherder
+
+  - app_name: firefox_desktop_background_tasks
+    canonical_app_name: Firefox Desktop background tasks
+    app_description: |
+      Background tasks of Firefox Desktop.
+      Currently used by `BackgroundTask_removeDirectory`.
+    url: https://github.com/mozilla/gecko-dev
+    notification_emails:
+      - krosylight@mozilla.com
```

Comment:
Maybe ask :nalexander if there are other emails/lists to include?

Reply:
asking krosylight in the bug right now